src/content/en/fundamentals/app-install-banners/index.md

```diff
@@ -29,7 +29,7 @@ has <code>"prefer_related_applications": true</code>, the
 <a href="/web/fundamentals/app-install-banners/native">native app install
 prompt</a> will be shown instead.
 
-## Show the add to home screen dialog {: #trigger }
+## Show the Add to Home Screen dialog {: #trigger }
```

Comment:
Last I heard from the Android writers 'home screen' is lower case. They may or may not be successful at enforcing that.

Reply:
I was just being consistent with the rest of the doc, for example [lines 35 & 39](https://github.com/google/WebFundamentals/blame/master/src/content/en/fundamentals/app-install-banners/index.md#L35-L39). Feel free to lowercase all occurrences!